spotlight-client/src/NarrativeFooter/NarrativeFooter.tsx

```diff
@@ -0,0 +1,151 @@
+
+import { Link } from "@reach/router";
+import { observer } from "mobx-react-lite";
+import { rem } from "polished";
+import React from "react";
+import { animated, useSprings } from "react-spring/web.cjs";
+import styled from "styled-components/macro";
+import { SystemNarrativeTypeIdList } from "../contentApi/types";
+import getUrlForResource from "../routerUtils/getUrlForResource";
+import { useDataStore } from "../StoreProvider";
+import { colors, typefaces } from "../UiLibrary";
+import Arrow from "../UiLibrary/Arrow";
+
+const Container = styled.nav`
+  padding: ${rem(120)} ${rem(32)};
+
+  .NarrativeFooter__BackLink {
+    color: ${colors.link};
+    display: inline-block;
+    font-weight: 500;
+    font-size: ${rem(18)};
+    line-height: 1.7;
+    margin-top: ${rem(48)};
+    text-decoration: none;
+  }
+`;
+
+const Heading = styled.h2`
+  font-family: ${typefaces.display};
+  font-size: ${rem(32)};
+  line-height: 1.75;
+  letter-spacing: -0.04em;
+`;
+
+const LinkList = styled.ul`
+  display: flex;
+  font-size: ${rem(24)};
+  line-height: 1.5;
+  margin: ${rem(48)} -${rem(16)};
+`;
+
+const LinkListItem = styled.li`
+  border-top: 1px solid ${colors.rule};
+  flex: 1 1 auto;
+  margin: 0 ${rem(16)};
+
+  a {
+    color: ${colors.text};
+    display: block;
+    padding-top: ${rem(32)};
+    text-decoration: none;
+    width: 100%;
+  }
+`;
+
+const Footer: React.FC = () => {
+  const {
+    tenant,
+    tenantStore: { currentNarrativeTypeId },
+  } = useDataStore();
+
+  const narrativesToDisplay = SystemNarrativeTypeIdList.filter(
+    (id) => id !== currentNarrativeTypeId
+  );
+
+  const [arrowSprings, setArrowSprings] = useSprings(
+    narrativesToDisplay.length,
+    () => ({
+      opacity: 0,
+      from: { opacity: 0 },
+    })
+  );
+
+  if (!tenant) return null;
+
+  function transitionArrow({
+    index,
+    visible,
+  }: {
+    index: number;
+    visible: boolean;
+  }) {
+    return () => {
+      // @ts-expect-error type error in current version,
+      // https://github.com/pmndrs/react-spring/issues/861
+      setArrowSprings((springIndex: number) => {
+        if (springIndex !== index) return;
+        return { opacity: visible ? 1 : 0 };
+      });
+    };
+  }
+
+  return (
+    <Container aria-label="collections">
+      <Heading>Continue Reading</Heading>
+      <LinkList>
+        {narrativesToDisplay.map((id, index) => {
+          const narrative = tenant.systemNarratives[id];
+          if (narrative)
+            return (
+              <LinkListItem key={id}>
+                <Link
+                  to={getUrlForResource({
+                    page: "narrative",
+                    params: { tenantId: tenant.id, narrativeTypeId: id },
+                  })}
+                  onMouseOver={transitionArrow({ index, visible: true })}
+                  onFocus={transitionArrow({ index, visible: true })}
+                  onMouseOut={transitionArrow({ index, visible: false })}
```

Comment:
In IE11 I found that one of the footer link arrows remained visible after clicking through to a different narrative. Was able to reproduce with:

1. From the Sentencing page, scroll to the bottom of the page to see the Narrative Footer
2. Click on the Probation footer link
3. Scroll to the bottom of the page and see that the Prison arrow is still visible, even clicking elsewhere on the page.

Reply:
oh good catch! I think I see the issue